### PR TITLE
Ext stm32cube readme fix

### PR DESCRIPTION
**Pull request by erwango**
To comply with Zephyr Contributing guidelines regarding
ext/ components, update README for each STM32 series

### CODE REVIEW

File: ext/hal/st/stm32cube/stm32f0xx/README [hunk 1]
```diff
@@ -1,15 +1,48 @@
...
+   It is composed of STM32Cube hardware abstraction layer (HAL) and low layer (LL) plus a set
+   of CMSIS headers files, one for each SoC in STM32F0 series.
+
+Dependencies:
+    None.
+
+URL:
+   http://www.st.com/en/embedded-software/stm32cubef0.html
+
+commit:
```
Comment (ydamigos) [inline ext/hal/st/stm32cube/stm32f0xx/README]: Does this make sense in stm32cubefx case? The project is not hosted on a git repository or something similar.

Comment (erwango) [reply ext/hal/st/stm32cube/stm32f0xx/README]: Not much but copy/pasted from similar library (with also absence of git repo),
and README reported form that library was reported compliant with Zephyr guidelines,
hence I reused it to minimize chances to get a comment to rework it.

**Review by ydamigos — APPROVED** [review]

**Comment by ydamigos** [comment]
Commit messages' header needs a space between `ext:hal:`.

**Review by galak — APPROVED** [review]

**Review by carlescufi — COMMENTED** [review]
LGTM

**Pull request merged**